pyproject.toml

```diff
@@ -9,7 +9,7 @@ requires-python = ">=3.8"
 dependencies = [
     'docutils', 'importlib-metadata',
     'numpy', 'scipy', 'matplotlib', 'pdoc',
-    'pytest', 'pytest-mpi>=0.6'
+    'pytest', 'pytest-mpi>=0.6', 'pylint'
```

Comment:
i dont think we need this as a requirement. This is only needed for the CI. agree?

Reply:
Yes; should I just `pip install pylint` as a step in CI?